core/dronecore.cpp

```diff
@@ -154,6 +230,8 @@ const char *DroneCore::connection_result_str(ConnectionResult result)
             return "Destination IP unknown";
         case ConnectionResult::CONNECTIONS_EXHAUSTED:
             return "Connections exhausted";
+        case ConnectionResult::CONNECTION_URL_INVALID:
+            return "Invalid Connection url";
```

Comment:
`URL` would be nicer, I think.

Reply:
"Connection" is not a proper noun, so should not be capitalised.

This should be "Invalid connection URL"